src/EnergyPlus/InputProcessing/InputValidation.cc

```diff
@@ -79,6 +79,26 @@ std::vector<std::string> const &Validation::warnings()
     return warnings_;
 }
 
+// this is a little bit risky, since it's theoretically
+// possible that the schema could change?
+// But we're trying it to see the impact on the code
+const valijson::Schema &validation_schema(const json *schema)
+{
+    [[maybe_unused]] static const json *last_schema = schema;
+
+    assert(last_schema == schema);
+
+    static const std::unique_ptr<valijson::Schema> retval = [&]() {
+        auto vs = std::make_unique<valijson::Schema>();
+        valijson::SchemaParser parser;
+        valijson::adapters::NlohmannJsonAdapter schema_doc(*schema);
+        parser.populateSchema(schema_doc, *vs);
+        return vs;
+    }();
+
+    return *retval;
+}
```

Comment:
If the schema could change while EnergyPlus is running, I think we'd have plenty of other things to worry about changing.  I don't mind that you are doing extra protection here as an assert, but I don't think that's something to sweat here.